pkg/api/types.go

```diff
@@ -438,6 +440,21 @@ type GlusterfsVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty"`
 }
 
+// CephfsVolumeSource represents a Cephfs Mount that lasts the lifetime of a pod
+type CephfsVolumeSource struct {
+	// Required: EndpointsName is the endpoint name that represents the Ceph monitors
+	EndpointsName string `json:"endpoints"`
```

Comment:
still open comment - shouldn't this be a LocalObjectRef ?  @pmorie that is what we decided, right?